Casks/font/font-f/font-formudpgothic.rb

```diff
@@ -0,0 +1,13 @@
+cask "font-formudpgothic" do
+  version "1.05101"
+  sha256 "6fe94947faad0dec2cab535d1a927d4f5f8ebe9231dbe4fd57bfa42646fce2f5"
+
+  url "https://github.com/jz5/FORM-UDPGothic/releases/download/v#{version}/FORMUDPGothic.zip"
+  name "FORMUDPGothic"
```

Comment:
```suggestion
  name "FORM UDPGothic"
  name "帳票 UDPゴシック"
```

Based on the upstream repository, there should be a space in the name. We can also add the Japanese name.